man/ostree-summary.xml

```diff
@@ -73,7 +73,14 @@ Boston, MA 02111-1307, USA.
                 <term><option>-u</option></term>
 
                 <listitem><para>
-		  Update the summary file.
+                  Update the summary file.
+                </para><para>
+                  Any additional arguments to the command
+                  are treated as additional key–value pairs to be added to the
+                  summary file as additional metadata. They must be in the format
+                  <command><replaceable>KEY</replaceable>=<replaceable>VALUE</replaceable></command>
+                  or as two separate arguments. The values must be in GVariant
+                  text format.
```

Comment:
Suggest prefixing custom values with something like `exampleos.` to avoid clashes with the `ostree.` prefix?